openshift/github-mirror.yaml

```diff
@@ -137,9 +137,9 @@ objects:
 
 parameters:
 - name: IMAGE
-  value: quay.io/app-sre/github-mirror
+  value: quay.io/redhat-services-prod/app-sre-tenant/github-mirror-master/github-mirror-master
```

Comment:
Is there any way to grant our quay accounts access to this image or make it public like before?

Reply:
Fixed

Reply:
what's the difference between `redhat-user-workloads` and `redhat-services-prod`?